draft-ietf-quic-multipath.md

```diff
@@ -1399,6 +1399,17 @@ it cannot allocate sufficient resources to maintain it. This is due to
 limitations in the transport layer's capacity. This error indicates that
 resource constraints prevent the continuation of the path.
 
+UNSTABLE_INTERFACE (TBD-11): The endpoint is abandoning the path because
+the used interface is considered to be unstable. This conition can occur, e.g., 
+due to a weak wireless signal or frequent mobility events.
+
+NO_CID_AVAILABLE (TBD-12): The endpoint is abandoning the path due to
+the lack of a connection ID for this path.
+This may occur when the peer initiates a new path
+but has not provided a corresponding connection ID for the path ID
+(or the packet containing the connection IDs has not arrived yet).
```

Comment:
Should this error position itself in relation to the PATH_CID_BLOCKED in #468. Commonly a endpoint would first send PATH_CID_BLOCKED to indicate the lack of CID, and eventually it might decide to close the path using PATH_ABANDON with the NO_CID_AVAILABLE error code.

Reply:
Yes, that's true. But we need to merge #468 first or we'll have an incorrect reference, and we also need to change the error code TBD-number at that point. I'd like to change these two parts after #468 is merged.

Reply:
Not sure if an endpoint would typically do that. It's is equally valid to just abandon the path right away and not have an half-open path dangling around. I'm not sure we want to give any guidance here at all.

Reply:
After some additional thought I think this is fine. Implementers will likely figure out what I wrote, or if not even that will be clear for the ones receiving PATH_ABANDON with this error code.